controls/cf_agent.cf

```diff
@@ -26,8 +26,11 @@ body agent control
       skipidentify => "true";
 
       # explicitly not supported (but they should break long before)
-      abortclasses => { "cfengine_3_3", "cfengine_3_4" };
+      abortclasses => { "cfengine_3_3", "cfengine_3_4", @(def.control_agent_abortclasses) };
 
+      # The abortbundleclasses slist contains regular expressions that match
+      # classes which if defined lead to termination of current bundle.
+      abortbundleclasses => { @(def.control_agent_abortbundleclasses) };
```

Comment:
@nickanderson I don't understand why, it looks correct to me, but this gives us an error when trying to bootstrap (see core master travis).